electron-app/src/websites/artconomy/artconomy.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { DefaultFileOptions } from '../../submission/submission-part/interfaces/default-options.interface';
+
+export interface ArtconomyOptions extends DefaultFileOptions {
```

Comment:
ArtconomyFileOptions as it extends DefaultFileOptions.

This hasn't been a super consistent pattern in all of the code since I decided to be lazy early on in favor of speed but going forward this is the standard.